Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - Support gamepad vibration on Switch.
+- Support for Bluetooth Xbox One controllers on macOS.
```

Comment:
The usual conflict around CHANGELOG.md...

Reply:
Needs to go up there in the 0.3-preview section.

Reply:
Ah, yes. I just merged in github and did not even notice this.